web/src/components/dashboard/SidebarCardsContainer.test.tsx

```diff
@@ -77,6 +77,51 @@ describe("<SidebarCardsContainer />", () => {
   };
 
   describe("nudges", () => {
+    it("renders content for visible card", () => {
+      const sidebarCards = {
+        welcome: generateSidebarCardContent({ contentMd: "WelcomeCardContent" }),
```

Comment:
[pebble] could we name this fake card or test card, to decouple it a realistic card like we did below?

Reply:
done